designate.md

```diff
@@ -0,0 +1,36 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
+and records over the network using pools of DNS servers. From an Admin's perspective there is not
+much to configure as the default for deployment are well in place. Neutron needs some additional
+settings for integration with desginate[1], the barclamp currently does not configure these but if
+required the admin can enable these as required.
+
```

Comment:
Maybe go into some more details as to how one would do that. Can the user just add a drop in configuration file for Neutron? Or are there hidden settings in the Neutron barclamp that the user can use?